contribs/commercialTrafficApplications/src/main/java/org/matsim/contrib/commercialTrafficApplications/jointDemand/commercialJob/JointDemandConfigGroup.java

```diff
@@ -18,128 +18,142 @@
-    public double getMinJobScore() {
-        return minJobScore;
-    }
-
-    /**
-     * @param minJobScore --{@value #MINJOBSCOREDESC}
-     */
-//    @StringSetter(MINJOBSCORE)
-    public void setMinJobScore(double minJobScore) {
-        this.minJobScore = minJobScore;
-    }
-
-    @Override
-    public Map<String, String> getComments() {
-        Map<String, String> map = super.getComments();
-        map.put(FIRSTLEGBUFFER, FIRSTLEGBUFFERDESC);
-        map.put(MAXJOBSCORE, MAXJOBSCOREDESC);
-        map.put(MINJOBSCORE, MINJOBSCOREDESC);
-        map.put(ZEROUTILDELAY, ZEROUTILDELAYDESC);
-        return map;
-    }
-
-    @Override
-    protected void checkConsistency(Config config) {
-        super.checkConsistency(config);
-        if (getMaxJobScore() < getMinJobScore()) {
-            throw new RuntimeException("Minimum Score for commercial jobs is higher than maximum score");
-        }
-    }
+	@Positive
+	private double firstLegTraveltimeBufferFactor = 2.0;
+	public static final String FIRSTLEGBUFFER = "firstLegBufferFactor";
+	private static final String FIRSTLEGBUFFERDESC = "Buffer travel time factor for the first leg of a freight tour.";
+
+	public static final String MAXJOBSCORE = "maxJobScore";
+
+	@Positive
+	private double zeroUtilityDelay = 1800;
+	public static final String ZEROUTILDELAY = "zeroUtilityDelay";
+	private static final String ZEROUTILDELAYDESC = "Delay (in seconds) that marks the threshold for zero utility";
+	public static final String MINJOBSCORE = "minJobScore";
+	private static final String MAXJOBSCOREDESC = "Score for performing job in time.";
+	private static final String MINJOBSCOREDESC = "Minimum score for delayed commercial jobs. "
+			+ "Note that if the customer is not served at all, that creates a score of zero. "
+			+ "So if this value is set to negative, that means an intensively delayed job is worse than no job.";
+	@Positive
+	private double maxJobScore = 6;
+	private double minJobScore = -6;
+
+	public static final String KEEPUSEDOPERATORSINTERVAL = "keepUsedOperatorsInterval";
+	public static final String KEEPUSEDOPERATORSINTERVALDESC = ChangeCommercialJobOperator.SELECTOR_NAME
+			+ " is actively used only every n-th iteration. Between this interval, assigned operator "
+			+ "per job is kept constant and jsprit tourplanning gets bypassed.";
+	@PositiveOrZero
+	private int keepUsedOperatorsInterval = 0;
+
+	public static final String GROUP_NAME = "commercialTraffic";
+
+	public JointDemandConfigGroup() {
+		super(GROUP_NAME);
+	}
+
+	public static JointDemandConfigGroup get(Config config) {
+		return (JointDemandConfigGroup) config.getModules().get(GROUP_NAME);
+	}
+
+	/**
+	 * @return -- {@value #KEEPUSEDCARRIERINTERVAL}
+	 */
+	public int getKeepUsedOperatorsInterval() {
+		return keepUsedOperatorsInterval;
+	}
+
+	/**
+	 * @param -- {@value #KEEPUSEDCARRIERINTERVAL}
+	 */
+	public void setKeepUsedOperatorsInterval(int interval) {
+		this.keepUsedOperatorsInterval = interval;
```

Comment:
i think i would prefer something like setChangeCommercialJobOperatorInterval (something that is directly related to the strategy name)

Reply:
Changed to setChangeCommercialJobOperatorInterval